docs/libu64.md

```diff
@@ -0,0 +1,73 @@
+# libu64
+
+This document explains the reasoning behind discovering libu64 files.
+
+libu64 is known about from the Animal Crossing (GameCube) map file.
```

Comment:
"the" implies only one but there's several different "Animal Crossing (GameCube)" games and several map files for each. Maybe spell out "Doubutsu no Mori+" instead?